src/backend/marsha/core/permissions.py

```diff
@@ -410,7 +410,11 @@ def has_permission(self, request, view):
         the current logged in user is one of the administrators of the organization to which
         this video's playlist belongs.
         """
-        video_id = request.data.get("video") or request.query_params.get("video")
+        video_id = (
+            request.data.get("video")
+            or request.query_params.get("video")
+            or view.kwargs.get("video_id")
```

Comment:
Should not be first ? If there is an order I think this one should be first. This what you do in the `get_related_video_id` function. WDYT ?

Reply:
You are true, and I think should directly use the "view.get_related_video_id", every views that need this permission inherit from ObjectVideoRelatedMixin (this pattern has already been done for the "IsTokenResourceRouteObjectRelatedVideo" permission).